### ISSUE TITLE
'NoneType' object has no attribute 'utils'

### ISSUE THREAD

**Issue by glane2609** [amaiya/ktrain#473]
The code was running fine for 0.30.0 , now I'm getting following error

![image](https://user-images.githubusercontent.com/62784645/212056059-90946f0b-f740-44b0-bebb-f147ac6b6ed0.png)

`import ktrain
from ktrain import text
from ktrain.text import texts_from_array
from sklearn.model_selection import train_test_split

classes =  list(set(reviews_df.score))

X_train, X_test, y_train, y_test = train_test_split(reviews_df.content, reviews_df.score, test_size=0.2, random_state=42)

# Pre Processing 
trn, val, preproc = text.texts_from_array(x_train=X_train.values, y_train=y_train.values,
                                          x_test=X_test.values, y_test=y_test.values,
                                                                   class_names=classes,
                                                                   val_pct=0.1, 
                                                                   preprocess_mode='distilbert')

# Model 
model = text.text_classifier('distilbert', train_data=trn, preproc=preproc)`

**Comment by amaiya** [comment]
I cannot reproduce the problem you're describing.  As shown in [this Google colab notebook](https://colab.research.google.com/drive/1E0nJSZDDCTH3_Zxck4EliStrxK5JZwWt?usp=sharing), everything seems to work fine with the latest version of **ktrain**.

You might try re-creating your virtual environment from scratch in case the problem is with a dependency (e.g., `pandas`).

For further assistance, please share a Google Colab notebook that reproduces the problem you're seeing exactly.

**Comment by glane2609** [comment]
Hey mate,

Thank you so much , it works..
Can you help me in reducing the epoch ETA, its showing more around 90 mins

**Comment by amaiya** [comment]
Great.  First step to reduce ETA would be to make sure you're using a GPU with `nvidia-smi`.

**Comment by glane2609** [comment]
can you send me a link, so I can download it?

**Comment by amaiya** [comment]
[GPU setup section](https://www.tensorflow.org/install/pip)

**Comment by glane2609** [comment]
[Glane.zip](https://github.com/amaiya/ktrain/files/10416411/Glane.zip)


Hello , can you help me run this. Sadly I don't have Nvidia to install drivers. I have done oversampling. I wanna make sure the accuracy part goes to around 70%. Feel free to make changes , I have also attached data , so you dont have to scrape again. Hoping for a positive response
